Kitodo/src/main/java/org/kitodo/production/services/data/RulesetService.java

```diff
@@ -186,10 +176,8 @@ List<Map<String, Object>> findByClientId(Integer clientId) throws DataException
      * @return search result
      */
     public Map<String, Object> findByTitleAndFile(String title, String file) throws DataException {
-        BoolQueryBuilder query = new BoolQueryBuilder();
-        query.must(createSimpleQuery(RulesetTypeField.TITLE.getKey(), title, true, Operator.AND));
-        query.must(createSimpleQuery(RulesetTypeField.FILE.getKey(), file, true, Operator.AND));
-        return findDocument(query);
+        // TODO delete method stub
+        throw new UnsupportedOperationException("no longer used function");
```

Comment:
Method is not marked as deprecated but is throwing this exception? Why not remove this method and maybe adjust the still used usage places?